test/testfixture/recipe_funcs.go

```diff
@@ -0,0 +1,94 @@
+package testfixture
+
+import (
+	errs "github.com/pkg/errors"
+)
+
+// A RecipeFunction tells the test fixture to create n objects of a given kind.
+// You can pass in customize-entity-functions in order to manipulate the objects
+// before they get created.
+type RecipeFunction func(fxt *TestFixture) error
+
+const checkStr = "expected at least %d \"%s\" objects but found only %d"
+
+func (fxt *TestFixture) deps(fns ...RecipeFunction) error {
+	if !fxt.isolatedCreation {
+		for _, fn := range fns {
+			if err := fn(fxt); err != nil {
+				return errs.Wrap(err, "failed to setup dependency")
+			}
+		}
+	}
+	return nil
+}
+
+// CustomizeTenantFunc is directly compatible with CustomizeEntityFunc
+// but it can only be used for the Tenants() recipe-function.
+type CustomizeTenantFunc CustomizeEntityFunc
+
+// Tenants tells the test fixture to create at least n identity objects.
+//
+// If called multiple times with differently n's, the biggest n wins. All
+// customize-entitiy-functions fns from all calls will be respected when
+// creating the test fixture.
+//
+// Here's an example how you can create 42 identites and give them a numbered
```

Comment:
I guess, this example is not applicable here and needs a rewrite, because you don't have `Identities` as a recipe function, do you?